docs/source/services/timelock_service/migration.rst

```diff
@@ -13,12 +13,37 @@ Otherwise, this can lead to serious data corruption.
 
 Automated Migration
 -------------------
+
 .. warning::
     Automated migrations are only implemented for Cassandra.
 
 1. If you are using the Casssandra key value service and have added the :ref:`Timelock client configuration <timelock-client-configuration>`, then starting/re-starting the service will automatically migrate the service.
 2. If you are using any other KVS, please follow the instructions at :ref:`manual-timelock-migration`.
 
+Verification
+~~~~~~~~~~~~
+
+It may be useful to verify that an automatic migration was carried out successfully. In addition to simply performing
+smoke tests of your clients and checking that they still work, there are a few other ways of checking that the
+migration occurred correctly.
+
+1. **Request Logs**: Find the TimeLock leader, and search its request logs for calls to the ``fastForwardTimestamp``
+   endpoint. These should be called with a ``currentTimestamp`` query parameter, which should be the bound in the
+   embedded timestamp bound store before the migration.
```

Comment:
Unfortunately it's not easy to get after the upgrade; the best I could come up with was searching for the most recent instance of `[CAS] Setting cached limit to {}.` in the logs